configuration/configmgr/configmgrlib/mod_template_support/Variable.cpp

```diff
@@ -82,14 +81,14 @@ std::string Input::getValue(size_t idx) const
 }
 
 
-void Input::setValue(const std::string &value)
+void Variable::addValue(const std::string &value)
 {
-    if (!m_values.empty())
-    {
-        m_values[0] = value;
-    }
-    else
-    {
+//    if (!m_values.empty())
```

Comment:
Any reason to keep the commented out code?

Reply:
Um, no. I thought I had removed all the commented code, guess I missed this one. I will remove.